documentation/course/e2/README.md

```diff
@@ -1798,6 +1908,29 @@ TODO @tchojnacki: Dodać diagram pakietów, opis architektury.
 
 TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
+### API
+
+#### API publiczne
+
+| **Rola**    | **Metoda** | **Endpoint**                   | **Wymagania** | **Opis**                                      |
+| ----------- | ---------- | ------------------------------ | ------------- | --------------------------------------------- |
+| `passenger` | `GET`      | `/ext/v1/offers`               | TODO          | Pobranie listy dostępnych ofert biletowych.   |
+| `passenger` | `GET`      | `/ext/v1/offers/:id`           | TODO          | Pobranie informacji o ofercie biletu.         |
+| `passenger` | `GET`      | `/ext/v1/tickets`              | TODO          | Pobranie listy zakupionych biletów.           |
+| `passenger` | `POST`     | `/ext/v1/tickets`              | TODO          | Zakup biletu.                                 |
+| `passenger` | `GET`      | `/ext/v1/tickets/:id`          | TODO          | Pobranie informacji o bilecie.                |
+| `passenger` | `POST`     | `/ext/v1/tickets/:id/validate` | TODO          | Walidacja biletu.                             |
+| `inspector` | `POST`     | `/ext/v1/tickets/:id/inspect`  | TODO          | Inspekcja biletu.                             |
```

Comment:
```suggestion
| `inspector` | `POST`     | `/ext/v1/tickets/:id/inspect`  | TODO          | Sprawdzenie ważności biletu.                             |
```
per dictionary

(if you commit my suggestions please reformat the file, I didn't change whitespace count to perserve column width)